validator-cli/src/ArbToEth/watcherArbToGnosis.ts

```diff
@@ -755,58 +746,58 @@ async function getClaimForEpoch(
   return null;
 }
 
-async function calculateDisputeResolutionProfitability(
-  epoch: number,
-  claim: ClaimStruct,
-  veaOutbox: VeaOutboxArbToGnosis,
-  veaInbox: VeaInboxArbToGnosis,
-  providerGnosis: JsonRpcProvider,
-  providerArb: JsonRpcProvider,
-  providerEth: JsonRpcProvider
-): Promise<{ profitable: boolean; estimatedProfit: BigNumber }> {
-  try {
-    const deposit = (await retryOperation(() => veaOutbox.deposit(), 1000, 10)) as BigNumber;
-    const totalReward = deposit;
-    const minimumProfit = totalReward.mul(40).div(100); // 40% of total reward
-    let maximumAllowableCost = totalReward.sub(minimumProfit);
-    let totalCost = BigNumber.from(0);
-
-    // 1. Costs on Gnosis Chain
-    const gnosisGasEstimate = await veaOutbox.estimateGas.challenge(epoch, claim);
-
-    const gnosisGasPrice = await providerGnosis.getGasPrice();
-    const gnosisCost = gnosisGasEstimate.mul(gnosisGasPrice);
-
-    if (gnosisCost.gt(maximumAllowableCost)) {
-      return { profitable: false, estimatedProfit: constants.Zero };
-    }
-    totalCost = totalCost.add(gnosisCost);
-    maximumAllowableCost = maximumAllowableCost.sub(gnosisCost);
-
-    const l2Network = await getArbitrumNetwork(providerArb);
-
-    const arbGasEstimate = (await retryOperation(
-      () => veaInbox.estimateGas.sendSnapshot(epoch, 200000, claim),
-      1000,
-      10
-    )) as BigNumber;
-
-    const arbGasPrice = (await retryOperation(() => providerArb.getGasPrice(), 1000, 10)) as BigNumber;
-    const arbCost = arbGasEstimate.mul(arbGasPrice);
-
-    if (arbCost.gt(maximumAllowableCost)) {
-      return { profitable: false, estimatedProfit: constants.Zero };
-    }
-    totalCost = totalCost.add(arbCost);
-    maximumAllowableCost = maximumAllowableCost.sub(arbCost);
-
-    // 3. Costs on Ethereum (for Arbitrum -> Ethereum message)
-    //TODO : L2 to L1 message execution gas cost
-  } catch (error) {
-    console.error("Error calculating profitability:", error);
-    return { profitable: false, estimatedProfit: constants.Zero };
-  }
-}
+// async function calculateDisputeResolutionProfitability(
```

Comment:
If the code is not executing, just remove it. If we want it in the future we can always go back in the git history.